docs/elasticsearch-spec.asciidoc

```diff
@@ -50,6 +50,43 @@ spec:
 
 For more information on Elasticsearch settings, see https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html[Configuring Elasticsearch].
 
+[id="{p}-volume-claim-templates"]
+=== Volume Claim Templates
+
+By default the operator will create a `PVC` with a capacity of 1Gi for every Pod in an Elasticsearch cluster. This is to ensure that there is no data loss if a Pod is deleted.
```

Comment:
I would tend to use the present rather than the future. `s|will create|creates|`